6-anthos-install/README.md

```diff
@@ -1,5 +1,9 @@
 # Deploying Bank of Anthos
-These instructions need to be run on the bastion host. To access the bastion host, open the Google Cloud console, and navigate to Compute Engine in the GKE project `prj-bu1-d-boa-gke`. Then, select the `gce-bastion-us-west1-b-01` and click on `ssh`. You need to be a whitelisted member to access this node which can be achieved either [manually](https://cloud.google.com/iap/docs/using-tcp-forwarding#grant-permission) or by adding your GCP User Identity in the list of `bastion_members` in `5-infrastructure/business_unit_1/<environment>/main.tf`
+
+These instructions need to be run on the bastion host.
```

Comment:
Can we make this a h3/h4 so it wont be missed with other txt?

Reply:
Layout changed to highlight the instructions